docs/src/components/app/index.js

```diff
@@ -0,0 +1,344 @@
+// Dependencies
+import { h, Component } from "preact";
+import Router from "preact-router";
+
+// Material Components
+import Formfield from "preact-material-components/Formfield";
+import LayoutGrid from "preact-material-components/LayoutGrid";
+import Switch from "preact-material-components/Switch";
+import Toolbar from "preact-material-components/Toolbar";
+
+// Routes
+import ButtonPage from "../../routes/button";
+import CardPage from "../../routes/card";
+import CheckboxPage from "../../routes/checkbox";
+import DialogPage from "../../routes/dialog";
+import DrawerPage from "../../routes/drawer";
+import ElevationPage from "../../routes/elevation";
+import FabPage from "../../routes/fab";
+import FormFieldPage from "../../routes/form-field";
+import HomePage from "../../routes/home";
+import IconPage from "../../routes/icon";
+import IconTogglePage from "../../routes/icon-toggle";
+import LayoutGridPage from "../../routes/layout-grid";
+import LinearProgressPage from "../../routes/linear-progress";
+import ListPage from "../../routes/list";
+import MenuPage from "../../routes/menu";
+import RadioPage from "../../routes/radio";
+import SelectPage from "../../routes/select";
+import SliderPage from "../../routes/slider";
+import SnackbarPage from "../../routes/snackbar";
+import SwitchPage from "../../routes/switch";
+import TabsPage from "../../routes/tabs";
+import TextfieldPage from "../../routes/textfield";
+import ToolbarPage from "../../routes/toolbar";
+
+// Components
+import Menu from "../menu";
+
+// Styles
+import "preact-material-components/style.css";
+import "./style";
+
+// Class
+export default class Home extends Component {
+  constructor() {
+    super();
+    this.state = {
+      darkMode: false
+    };
+    this.menuItems = [
+      {
```

Comment:
how about this, we add this info in `prerender.json` as a separate node, and import that json in this file and loop over it. will our code pretty cleaner

Reply:
Will check if that's doable. Can't quite have something like ```component: HomePage``` in JSON.

Reply:
ahhh ok